docs/src/examples/components/Icon/Variations/IconExampleColor.shorthand.tsx

```diff
@@ -1,13 +1,59 @@
 import React from 'react'
-import { Icon } from '@stardust-ui/react'
+import { Icon, Grid, Text } from '@stardust-ui/react'
 
 const IconExampleColor = () => (
-  <div>
-    <Icon name="home" />
-    <Icon name="home" variables={{ color: 'blue' }} />
-    <Icon name="home" variables={{ color: 'red' }} />
-    <Icon name="home" variables={{ color: 'orange' }} />
-  </div>
+  <Grid columns="repeat(4, auto)" styles={{ alignItems: 'center' }} variables={{ gridGap: '10px' }}>
+    <Text content="INHERITED COLOR:" weight="bold" />
+    <div style={{ color: 'violet' }}>
+      <Icon name="calendar" bordered />
+      <Icon name="call" bordered />
+      <Icon name="call-video" bordered />
+    </div>
+    <Text content="INHERITED COLOR FOR OUTLINED ICONS:" weight="bold" />
+    <div style={{ color: 'yellowgreen' }}>
+      <Icon name="calendar" bordered variables={{ outline: true }} />
+      <Icon name="call" bordered variables={{ outline: true }} />
+      <Icon name="call-video" bordered variables={{ outline: true }} />
+    </div>
+    <Text
+      content={
+        <span>
+          USING THE <code>color</code> VARIABLE:
+        </span>
+      }
+      weight="bold"
+    />
```

Comment:
In the future, try to minimize the number of lines of code in doc site examples:

```jsx
    <Text
      content={
        <span>
          USING THE <code>color</code> VARIABLE:
        </span>
      }
      weight="bold"
    />
```
```jsx
<Text weight="bold">USING THE <code>color</code> VARIABLE:</Text>
```